elements/src/components/pos-list/pos-list.tsx

```diff
@@ -36,15 +48,24 @@ export class PosList implements ResourceAware {
   }
 
   receiveResource = (resource: Thing) => {
-    this.items = [];
     if (this.rel) this.items = resource.relations(this.rel).flatMap(relation => relation.uris);
   };
 
+  receivePodOs = async (os: PodOS) => {
+    this.os = os;
+    os.store.observeFindMembers(this.ifTypeof, this.disconnected$).subscribe(value => (this.items = value));
```

Comment:
issue: use `takeUntil(disconnected$)` here or store the subscription and unsubscribe in `disconnectedCallback()`instead of passing `disconnected$`